bin/convert.sh

```diff
@@ -0,0 +1,198 @@
+            done
+            break
+    fi
+    # Parse current opt
+    while [ x"$OPT" != x"-" ] ; do
+            case "$OPT" in
+                  -h | --help )
+                          print_usage
+                          exit 0
+                          ;;
+                  --force-cleanup )
+                          forceCleanup=true
+                          ;;
+                  --html=* )
+                          htmlFolder="${OPT#*=}"
+                          shift
+                          ;;
+                  --html )
+                          htmlFolder="$2"
+                          shift
+                          ;;
+                  --md=* )
+                          markdownFolder="${OPT#*=}"
+                          shift
+                          ;;
+                  --md )
+                          markdownFolder="$2"
+                          shift
+                          ;;
+                  --git-branch=* )
+                          gitBranch="${OPT#*=}"
+                          shift
+                          ;;
+                  --git-branch )
+                          gitBranch="$2"
+                          shift
+                          ;;
+                  * )
+                          OTHER_OPTS="$OTHER_OPTS $OPT"
+                          break
+                          ;;
+            esac
+            # Check for multiple short options
+            # NOTICE: be sure to update this pattern to match valid options
+            NEXTOPT="${OPT#-[cfr]}" # try removing single short opt
+            if [ x"$OPT" != x"$NEXTOPT" ] ; then
+                    OPT="-$NEXTOPT"  # multiple short opts, keep going
+            else
+                    break  # long form, exit inner loop
+            fi
+    done
+    # move to the next param
+    shift
+done
+
+# set and trim the REPOSITORIES_LIST_PARAMETER containing the list of git repositories
+gitList=$(echo "${OTHER_OPTS//[[:space:]]/}")
+
+# check whether gitList parameter has been provided
+if [[ -z ${gitList} ]]; then
+    echo -e "\nYou need to provide the <REPOSITORIES_LIST_FILE> !!!\n"
+    exit -1
+fi
+
+# log configuration
+echo -e "\n-------------------------------------------------------------------------------------------------------"
+echo -e "*** Tool Configuration *** "
+echo -e "-------------------------------------------------------------------------------------------------------"
+echo "Markdown folder: ${markdownFolder}"
+echo "Html folder: ${htmlFolder}"
+echo "Git Repositories file: ${gitList}"
+echo "Git branch: ${gitBranch}"
+echo -e "-------------------------------------------------------------------------------------------------------"
+
+# download the list file if it is a HTTP(s) resource
+if [[ ! -z ${gitList} && ${gitList} =~ ^https?://.+  ]]; then
+    echo -e "Downloading list of repositories..."
+    remoteGitList=${gitList}
+    gitList="/tmp/remoteGitList.txt"
```

Comment:
Should be deleted at some point.  Would also be good to use `mktemp`.

Reply:
Fixed by https://github.com/phnmnl/php-phenomenal-portal-app-library/pull/3/commits/3066699f04a63699d3597bdb5db15d9d052d71f2 and https://github.com/phnmnl/php-phenomenal-portal-app-library/pull/3/commits/2faff892d0d1d0e8e1987588dcc472000698a971